inspector/src/inspector.ts

```diff
@@ -153,6 +160,8 @@ export class Inspector {
                 },
             });
             ReactDOM.render(sceneExplorerElement, this._SceneExplorerHost);
+            const popupElement = React.createElement(PersistentPopupHostComponent, {globalState: this._GlobalState});
+            ReactDOM.render(popupElement, this._PopupHost);

```

Comment:
is there a cleanup needed when a popup is closed? Or are all components being unmounted correctly (i.e. - is react mounting/unmounting process enough?).

Reply:
I added a cleanup on the same function that cleans up the Tabs component, I think that will be enough